### PR TITLE
Add default to true when creating a stocktake line from a stockline

### LINKED ISSUE #1057: Can't create stocktake lines
When trying to insert a stocktake line, an update mutation is sent instead - oopsies 😁

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #1057 

- Also an off-road `isProduction` bug that I added with a PR recently

### CODE REVIEW

File: packages/host/src/Host.tsx [hunk 1]
```diff
@@ -29,8 +29,8 @@ const queryClient = new QueryClient({
     queries: {
       // These are disabled during development because they're
       // annoying to have constantly refetching.
-      refetchOnWindowFocus: isProduction(),
-      retry: isProduction(),
+      refetchOnWindowFocus: EnvUtils.isProduction(),
+      retry: EnvUtils.isProduction(),
```
Comment (joshxg) [inline packages/host/src/Host.tsx:33]: Offroad bug fix from my merge

File: packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/hooks.ts [hunk 2]
```diff
@@ -62,8 +62,8 @@ const stockLineToDraftLine = (
 ): DraftStocktakeLine => {
   return {
     stocktakeId,
-    countThisLine: false,
-    isCreated: false,
+    countThisLine: true,
+    isCreated: true,
```
Comment (joshxg) [inline packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/hooks.ts:66]: default to `isCreated` for lines with a stocktake line. If the user doesn't want to count it, then `countThisLine` will be false.

using `countThisLine` as true is questionable. It means when you add an item, you're counting all the stock lines in the store by default.. 🤔

**Comment by joshxg** [comment]
At the moment, the lines are blank when they're not counted.. so that is a bit of a terrible user experience. Also there's a bug where counted stock lines are doubled up. I'll look at those separately - probably best to get this in

**Review by mark-prins — APPROVED** [review]
🎉  I have no idea why this is required.. but it fixes the problem 😁

**Comment by joshxg** [comment]
😁 

the `isCreated` / `isUpdated` flags determine which mutation to send later on and the `isCreated` flag wasn't being set

**Pull request merged**